### PR TITLE
Update video.json

### PR DESCRIPTION
**Pull request by eat-sleep-code**
Safari no longer supports `autoplay` attribute.

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Source?

File: features-json/video.json [hunk 1]
```diff
@@ -195,7 +195,7 @@
       "9.1":"y",
       "10":"y",
       "10.1":"y",
-      "11":"y",
+      "11":"a #3",
```
Comment (cvrebert) [inline features-json/video.json]: You haven't added a corresponding third note under `notes_by_num`

**Comment by cvrebert** [comment]
I presume this is in reference to https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/ , which is more nuanced than simply "no longer supported".

**Comment by eat-sleep-code** [comment]
@cvrebert Yes, that is a reference to the issue.   So partially supported?   Because for it to be truly fully supported it should work as listed here without additional code or a workaround:  https://www.w3.org/TR/2012/WD-html-markup-20121025/video.html

> Instructs the UA to automatically begin playback of the video as soon as it can do so without stopping.

**Comment by eat-sleep-code** [comment]
I wasn't looking to get into a discussion about whether use of autoplay was appropriate.   I was simply pointing out, that a note needs to be made that it does not work "out of the box" with Safari as it requires additional remediation outside of the video tag for it to work.

**Comment by cvrebert** [comment]
Sure, I absolutely agree there should be a note. I was just pointing out that Safari's behavior is still technically spec-compliant.

**Comment by Fyrd** [comment]
Have manually added a note for Safari's autoplay behavior, so closing this issue.

**Pull request closed without merging**